kubernetes/cmsweb/services/dbs-default.yaml

```diff
@@ -1,82 +1,82 @@
-kind: Service
-apiVersion: v1
-metadata:
-  name: dbs-default
-  namespace: dbs
-spec:
-  selector:
-    app: dbs-default
-  ports:
-    - port: 8250
-      targetPort: 8250
-      name: dbs-default
-    - port: 18250
-      targetPort: 18250
-      name: dbs-default-mon
-    - port: 19250
-      targetPort: 19250
-      name: dbs-d-cpy-mon
----
-kind: Deployment
-apiVersion: extensions/v1beta1
-metadata:
-  labels:
-    app: dbs-default
-  name: dbs-default
-  namespace: dbs
-spec:
-  selector:
-    matchLabels:
-      app: dbs-default
-  replicas: 1 #PROD# 2
-  template:
-    metadata:
-      labels:
-        app: dbs-default
-    spec:
-      containers:
-      - image: cmssw/dbs
-        name: dbs-default
-        ports:
-        - containerPort: 8250
-          protocol: TCP
+        kind: Service
+        apiVersion: v1
+        metadata:
           name: dbs-default
-        - containerPort: 18250
-          protocol: TCP
-          name: dbs-default-mon
-        - containerPort: 19250
-          protocol: TCP
-          name: dbs-d-cpy-mon
-        volumeMounts:
-        - name: proxy-secrets
-          mountPath: /etc/proxy
-          defaultMode: 256
-        - name: secrets
-          mountPath: /etc/secrets
-          defaultMode: 256
-        - name: hostkey
-          mountPath: /etc/grid-security/hostkey.pem
-          readOnly: true 
-        - name: hostcert
-          mountPath: /etc/grid-security/hostcert.pem
-          readOnly: true 
-#PROD#  - name: logs-cephfs
-#PROD#    mountPath: /data/srv/logs/dbs
-      volumes:
-      - name: proxy-secrets
-        secret:
-          secretName: proxy-secrets
-      - name: secrets
-        secret:
-          secretName: dbs-default-secrets
-      - name: hostkey
-        hostPath:
-            path: /etc/grid-security/hostkey.pem
-            type: File
-      - name: hostcert
-        hostPath:
-            path: /etc/grid-security/hostcert.pem
-            type: File
-#PROD#- name: logs-cephfs
-#PROD#  persistentVolumeClaim:
-#PROD#      claimName: logs-cephfs-claim
+          namespace: dbs
+        spec:
+          selector:
+            app: dbs-default
+          ports:
+            - port: 8250
+              targetPort: 8250
+              name: dbs-default
+            - port: 18250
+              targetPort: 18250
+              name: dbs-default-mon
+            - port: 19250
+              targetPort: 19250
+              name: dbs-d-cpy-mon
+        ---
```

Comment:
Muhammad, are you sure that your editor formatting made a right choice. I see that you indent entire file, e.g. at this line the selector shifted to the right by some (8) spaces. Please remove indentation.